src/components/Checkbox.js

```diff
@@ -75,17 +80,20 @@ class Checkbox extends React.Component {
 
     render() {
         return (
-            <Pressable
+            <PressableWithFeedback
                 disabled={this.props.disabled}
                 onPress={this.firePressHandlerOnClick}
                 onMouseDown={this.props.onMouseDown}
                 ref={this.props.forwardedRef}
+                onPressOut={this.onBlur}
```

Comment:
@Skalakid What is this `this.onBlur`? I always get undefined value for this?

Reply:
@Skalakid  any update on this?

Reply:
@abdulrahuman5196 Hey, sorry for the late response, I just fixed this by merging with the main ;) I think it was added after merging with main, now it's gone